packages/commercetools-api-request-builder/src/create-service.js

```diff
@@ -92,18 +92,27 @@ export default function createService (
     // Pass some options to further configure the URI:
     // - `projectKey`: will prefix the URI with the given projectKey
     build (options: BuildOptions = {}): string {
```

Comment:
@Siilwyn doesn't this have something to do with the error you were having this evening?